client/src/app/gateways/export/html-to-pdf.service/html-to-pdf.service.ts

```diff
@@ -21,17 +34,17 @@ export class HtmlToPdfService {
     /**
      * Normal line height for paragraphs
      */
-    private LINE_HEIGHT = 1.25;
+    protected LINE_HEIGHT = 1.25;
```

Comment:
`LINE_HEIGHT` can be overwritten within the `convertHtml` method. I would only expect constant values to be uppercase.